js/fizzbuzz.js

```diff
@@ -0,0 +1,70 @@
+function doFizzbuzz() {
+
+    // 入力された値を取得
+    var fizz = document.getElementById("fizz_num").value;

```

Comment:
昨今 `var` はほとんど使われません。
再代入が発生する場合は `let`、発生しない場合は `const` を利用してください。

```suggestion
    let fizz = document.getElementById("fizz_num").value;
```